clang_delta/RemoveUnusedFunction.cpp

```diff
@@ -424,7 +424,18 @@ SourceLocation RemoveUnusedFunction::getFunctionLocEnd(
                  const FunctionDecl *FD)
 {
   if (!FD->getDescribedFunctionTemplate())
-    return LocEnd;
+  {
```

Comment:
Can you change the coding style to:

    if (xxx) {
      ...
    }
    else {
      ...
    }